core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -147,6 +149,11 @@
     private boolean disableSetPropertiesSecurityCheckForCreateDdl;
     private boolean incrementalHashArrayLoadFactorEnabled = true;
 
+    private RetryPolicy retryPolicy = RetryPolicy.NONE;
+    private int retryAttempts = 4;
+    private Duration retryInitialDelay = new Duration(10, SECONDS);
```

Comment:
This might be too long for the default initial delay, especially for very short queries

Reply:
Setting it to something lower (<6s) might not give enough time for failure detector to detect a faulty node: https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/java/io/trino/failuredetector/FailureDetectorConfig.java#L30